test/dlc_tests/conftest.py

```diff
@@ -29,7 +30,11 @@
     "training",
     "inference",
     "gpu",
-    "cpu"
+    "cpu",
+    "eia",
+    "pytorch_eia",
+    "mxnet_eia",
+    "tensorflow_eia"
```

Comment:
@arjkesh @nskool Could you review the fixtures here? I think the fixture names will need to be a substring of the image uri. So `pytorch_eia`, `mxnet_eia`, `tensorflow_eia` will not work, afaik.

Reply:
Without adding these lines, eia test will get skipped.

Reply:
Just talked with @TusharKanekiDey -- the image URI is <account>.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-eia:2.0.0-cpu-py36-ubuntu18.04-pr-342, it should work as expected as long as the repository has <framework>-eia

Reply:
This goes against the convention we have in naming ECR repositories - The final release repositories have the repository name `<framework>-inference-eia`.

If we use `<framework>_eia` as the fixture name, the lookup comparison step will fail on images in all beta, gamma and release repositories.

Reply:
ah true, for production images this won't work then. are the PR repo naming conventions set in stone? or can they better mimic the final release repos?

Reply:
Not sure. There is a lot of code where we explicitly check for "training" or "inference", which seems to be lacking in the naming convention for EIA repositories.